exercises/resistor-color-trio/.meta/src/reference/java/ResistorColorTrio.java

```diff
@@ -0,0 +1,17 @@
+class ResistorColorTrio {
+    private enum Color {
+        black, brown, red, orange, yellow, green, blue, violet, grey, white;
+    };
```

Comment:
Java enums should generally be in UPPER_SNAKE_CASE.

I think the API should actually ingest the enum which should be a public file of its own (Color.java).

Reply:
Does the Java track exercise a policy or a preference wrt. multiple files handed out?

Reply:
I do not know of a specific policy, but I know that there are exercises which do this: https://github.com/exercism/java/tree/master/exercises/dominoes/src/main/java

There is one reference to multiple file submissions in the POLICIES file though (which implies it is totally fine):

> The first exercise in the track whose test suite mandates multiple solution files should be accompanied by a hints.md file reminding the practitioner that the CLI supports multiple file submissions.